mesh-otel/telemetry.yaml

```diff
@@ -5,15 +5,18 @@ mtls:
   backends:
     - name: ca-1
       type: builtin
-metrics:
-  enabledBackend: prometheus-1
-  backends:
-    - name: prometheus-1
-      type: prometheus
-      conf:
-        skipMTLS: true
-        envoy:
-          usedOnly: true
+---
+type: MeshMetric
+name: otel
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  default:
```

Comment:
should we use basic profile instead of these manual filters: https://github.com/kong/demo-scene/blob/a48ec6e0079601d340f79613549e1b2a4ea715a1/mesh-localityaware/k8s/otel-collectors.yaml#L165-L174 ?